package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "fastify-passport",
+  "version": "0.0.1",
+  "description": "Simple, unobtrusive authentication for Fastify.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "scripts": {
+    "build": "rm -rf ./build && mkdir build && tsc && git rev-parse HEAD > BUILD_SHA",
+    "lint": "eslint '*/**/*.{js,ts,tsx}'",
+    "lint:fix": "prettier --loglevel warn --write \"src/**/*.{ts,tsx}\" && eslint '*/**/*.{js,ts,tsx}' --quiet --fix",
+    "prepublishOnly": "yarn run build",
+    "test": "yarn run build && yarn run test:unit && yarn run lint",
```

Comment:
Can you use npm instead of yarn for the scripts? We are using npm everywhere in the ecosystem.